src/basal_mass_balance/BMB_main.f90

```diff
@@ -178,6 +208,11 @@ SUBROUTINE initialise_BMB_model( mesh, ice, BMB, region_name)
     SELECT CASE (choice_BMB_model)
       CASE ('uniform')
         ! No need to do anything
+      CASE ('prescribed')
+        CALL initialise_BMB_model_prescribed( mesh, BMB, region_name)
+      CASE ('prescribed_fixed')
```

Comment:
Do I understand it correctly that the difference is that 'prescribed' assumes the sub-grid scaling is already implied in the prescribed data? Maybe mention this somewhere.

Reply:
Nope. I added the following explanation to the header text of the BMB_prescribed module:

```   
  ! Prescribed BMB forcing. Read in a fixed 2D field of BMB values. At each time step, the applied BMB is determined by the chosen
  ! subgrid melt scheme. So in case of FCMP, for example, BMB_gr will be zero throughout the run.
  !
  ! If the BMB model = 'prescribed_fixed', the subgrid melt schem is only applied during initialisation. Hence, the integrated
  ! metric BMB_total will be constant throughout the run, regardless of GL migration or calving. GL advance will lead to nonzero
  ! BMB_gr, which may be useful to maintain the GL position. 
```